### ISSUE TITLE
[Structured List]: QA: Tooltip is not dismissible with esc key

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#9158]
**Engineering info:**  
- Innovation Team engineer: @andy-blum 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2528
- DPO consulting engineer: N/A

-----

### Description

Page element:
Tooltip content

Issue : Tooltip content is not dismissible with esc key.

Expected: Tooltip content should be dismissible with esc key.

User Impact: If the tooltip content is not dissmissible with esc key, motor impairment user will face difficulty in accessing the element.

### Component(s) impacted

Structured List
(With Complex content)

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by proeung** [comment]
Reference from the Carbon design system about the interaction - https://carbondesignsystem.com/components/tooltip/usage/#interactions

**Comment by proeung** [comment]
Blocked by https://github.com/carbon-design-system/carbon-web-components/issues/980

**Comment by andy-blum** [comment]
closing as duplicate of #8996
